react/filter-items-from-list/src/main.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Filteredlist from "./filteredlist";
+
+const App = () => {
+  return (
```

Comment:
No need to wrap a single element with a fragment, you can simply do `return <Filteredlist />`

Reply:
done